ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviderForm.tsx

```diff
@@ -482,53 +486,119 @@ function AuthProviderForm({
+                                                                value={attribute.attributeKey}
+                                                                onChange={onChange}
+                                                                isDisabled={isDisabled()}
+                                                            />
+                                                        </FormGroup>
+                                                        <FormGroup
+                                                            label="Value"
+                                                            fieldId={`requiredAttributes[${index}].attributeValue`}
+                                                        >
+                                                            <TextInput
+                                                                type="text"
+                                                                id={`requiredAttributes[${index}].attributeValue`}
+                                                                value={attribute.attributeValue}
+                                                                onChange={onChange}
+                                                                isDisabled={isDisabled()}
+                                                            />
+                                                        </FormGroup>
+                                                        {!isDisabled() && (
+                                                            <FlexItem>
+                                                                <Button
+                                                                    variant="plain"
+                                                                    aria-label="Delete required attribute"
+                                                                    style={{
+                                                                        transform:
+                                                                            'translate(0, 42px)',
+                                                                    }}
+                                                                    onClick={() =>
+                                                                        arrayHelpers.remove(index)
+                                                                    }
+                                                                >
+                                                                    <TrashIcon />
+                                                                </Button>
+                                                            </FlexItem>
+                                                        )}
+                                                        {!isUserResource(
+                                                            selectedAuthProvider.traits
```

Comment:
It's a tooltip in case user would like to delete the required attribute but is not allowed since the auth provider is declarative. We do similar thing for groups.